lib/tasks/test.rake

```diff
@@ -11,4 +11,6 @@ namespace :test do
 end
 
 # Ensure webpack files are compiled in case integration tests are executed
-Rake::Task[:test].enhance ['webpack:try_compile']
+Rake::Task[:test].enhance ['webpack:try_compile'] do
+  Rake::FileUtilsExt.verbose(false)
+end
```

Comment:
"Don't let Rake make file utils verbose in tests. Rake wraps file utils, making them verbose and hence printing their commands to stdout. Doing so adds a lot of noise to the test report, so we can set verbose to `false` in the context of the test task."

Reply:
I am testing the patch, I can't see the difference with and without this block. Can you paste me example output which is turned off with this patch? I still see webpack outputting everything as normal during test:

```
Child extract-text-webpack-plugin node_modules/extract-text-webpack-plugin/dist node_modules/css-loader/index.js!webpack/assets/javascripts/react_app/components/common/MessageBox/MessageBox.css:
       2 modules
Child extract-text-webpack-plugin node_modules/extract-text-webpack-plugin/dist node_modules/css-loader/index.js!webpack/assets/javascripts/react_app/common/commonStyles.css:
       [0] ./node_modules/css-loader!./webpack/assets/javascripts/react_app/common/commonStyles.css 201 bytes {0} [built]
        + 1 hidden module
```

Reply:
@lzap the block isn't intended to silence the webpack output - they are just two different prerequisites added to the test task and expressed in different ways. It will remove things like:

```
mkdir -p /tmp/migration_generator_test/core
rm -rf /tmp/migration_generator_test/core/../test_plugin
# etc....
```

(which you can see in the before/after you added)

Reply:
Oh I just realized this last night, thanks! Merging.